Issue 210 bug fix (#211)
np-guard/netpol-analyzer: README.md

Flags:
  -f, --file string            Write output to specified file
      --focusworkload       Focus connections of specified workload in the output (supported formats: <workload-name>, <workload-namespace>/<workload-name>)

>                             (for refining connections from Ingress/Route only, use `ingress-controller` as <workload-name>)

Reviewer: ```suggestion
                            (to focus connections from Ingress/Route only, use `ingress-controller` as <workload-name>)
```
